zone/client.cpp

```diff
@@ -6775,7 +6775,7 @@ void Client::UpdateClientXTarget(Client *c)
 // IT IS NOT SAFE TO CALL THIS IF IT'S NOT INITIAL AGGRO
 void Client::AddAutoXTarget(Mob *m, bool send)
 {
-	if (m->IsBot() || (m->IsPet() && m->IsPetOwnerBot())) {
+	if (m->IsBot() || (m->IsPet() && m->IsPetOwnerBot()) || (m->IsTempPet() && IsPetOwnerBot())) {
```

Comment:
Should be `m->IsPetOwnerBot()`, `IsPetOwnerBot()` implies that the current client calling `AddAutoXTarget` is a pet owned by a bot.

Reply:
Could also just do this.
```cpp
if (m->IsBot() || ((m->IsPet() || m->IsTempPet()) && m->IsPetOwnerBot()))
```